src/modules/components/odsModulesProvider/odsModulesProvider.tsx

```diff
@@ -37,20 +45,51 @@ export interface IOdsModulesProviderProps {
      * @see IOdsCoreContext
      */
     coreProviderValues?: IOdsCoreProviderProps['values'];
+    /**
+     * Context provider values.
+     */
+    values?: Partial<IOdsModulesContext>;
     /**
      * Children of the provider.
      */
     children?: ReactNode;
 }
 
+const odsModulesContextDefaults: IOdsModulesContext = {
+    copy: enCopy,
+};
+
+const odsModulesContext = createContext<IOdsModulesContext>(odsModulesContextDefaults);
+
 export const OdsModulesProvider: React.FC<IOdsModulesProviderProps> = (props) => {
-    const { queryClient = defaultQueryClient, wagmiConfig = defaultWagmiConfig, coreProviderValues, children } = props;
+    const {
+        queryClient = defaultQueryClient,
+        wagmiConfig = defaultWagmiConfig,
+        coreProviderValues,
+        children,
+        values,
+    } = props;
+
+    const contextValues = useMemo(
+        () => ({
+            copy: values?.copy ?? odsModulesContextDefaults.copy,
+        }),
+        [values],
+    );
 
     return (
-        <WagmiProvider config={wagmiConfig}>
-            <QueryClientProvider client={queryClient}>
-                <OdsCoreProvider values={coreProviderValues}>{children}</OdsCoreProvider>
-            </QueryClientProvider>
-        </WagmiProvider>
+        <odsModulesContext.Provider value={contextValues}>
+            <WagmiProvider config={wagmiConfig}>
+                <QueryClientProvider client={queryClient}>
+                    <OdsCoreProvider values={coreProviderValues}>{children}</OdsCoreProvider>
+                </QueryClientProvider>
+            </WagmiProvider>
+        </odsModulesContext.Provider>
     );
 };
+
+export const useOdsModulesContext = (): Required<IOdsModulesContext> => {
```

Comment:
The `required` is probably not needed, right?
```suggestion
export const useOdsModulesContext = (): IOdsModulesContext => {
```

Reply:
✅ https://github.com/aragon/ods/pull/221/commits/ba38b76702d85d1fdc5404a83db9d159de254e74